react-native/android/app/build.gradle

```diff
@@ -9,46 +12,65 @@ apply plugin: 'com.android.application'
-   jsBundleDirDebug: "$buildDir/intermediates/assets/debug",
+        // where to put the JS bundle asset in debug mode
+        jsBundleDirDebug: "$buildDir/intermediates/assets/debug",
 
-   // where to put the JS bundle asset in release mode
-   jsBundleDirRelease: "$buildDir/intermediates/assets/release",
+        // where to put the JS bundle asset in release mode
+        jsBundleDirRelease: "$buildDir/intermediates/assets/release",
 
-   // where to put drawable resources / React Native assets, e.g. the ones you use via
-   // require('./image.png')), in debug mode
-   resourcesDirDebug: "$buildDir/intermediates/res/merged/debug",
+        // where to put drawable resources / React Native assets, e.g. the ones you use via
+        // require('./image.png')), in debug mode
+        resourcesDirDebug: "$buildDir/intermediates/res/merged/debug",
 
-   // where to put drawable resources / React Native assets, e.g. the ones you use via
-   // require('./image.png')), in release mode
-   resourcesDirRelease: "$buildDir/intermediates/res/merged/release",
+        // where to put drawable resources / React Native assets, e.g. the ones you use via
+        // require('./image.png')), in release mode
+        resourcesDirRelease: "$buildDir/intermediates/res/merged/release",
 
-   // by default the gradle tasks are skipped if none of the JS files or assets change; this means
-   // that we don't look at files in android/ or ios/ to determine whether the tasks are up to
-   // date; if you have any other folders that you want to ignore for performance reasons (gradle
-   // indexes the entire tree), add them here. Alternatively, if you have JS files in android/
-   // for example, you might want to remove it from here.
-   inputExcludes: ["android/**", "ios/**"]
- ]
+        // by default the gradle tasks are skipped if none of the JS files or assets change; this means
+        // that we don't look at files in android/ or ios/ to determine whether the tasks are up to
+        // date; if you have any other folders that you want to ignore for performance reasons (gradle
+        // indexes the entire tree), add them here. Alternatively, if you have JS files in android/
+        // for example, you might want to remove it from here.
+        inputExcludes: ["android/**", "ios/**"]
+]
 
 apply from: "react.gradle"
 
+/**
+ * Set this to true to create two separate APKs instead of one:
+ *   - An APK that only works on ARM devices
+ *   - An APK that only works on x86 devices
+ * The advantage is the size of the APK is reduced by about 4MB.
+ * Upload all the APKs to the Play Store and people will download
+ * the correct one based on the CPU architecture of their device.
+ */
+def enableSeparateBuildPerCPUArchitecture = true
+
+/**
+ * Run Proguard to shrink the Java bytecode in release builds.
+ */
+def enableProguardInReleaseBuilds = false
+
 android {
     compileSdkVersion 23
     buildToolsVersion "23.0.1"
```

Comment:
`23.0.2` is the current version installed with the SDK, we might want to update that here.